templates.json

```diff
@@ -214,6 +214,11 @@
       "id": "forward-message-mailgun",
       "name": "Forward SMS message as an email (Mailgun)",
       "description": "The function will forward incoming SMS messages to an email address using the Mailgun API"
+    },
+    {
+      "id": "json-webhook",
+      "name": "JSON Webhook Adapter",
+      "description": "Convert a Twilio webhook event into a JSON webhook, suitable for services like Zapier and IFTTT"
```

Comment:
can we update the description here as well?

Reply:
Updated!